README.md

```diff
@@ -0,0 +1,118 @@
+# Java Function Invoker image:https://travis-ci.org/projectriff/java-function-invoker.svg?branch=master[Java Invoker, link=https://travis-ci.org/projectriff/java-function-invoker]
+
+## Install as a riff invoker
+
+```bash
+riff invokers apply -f java-invoker.yaml
+```
+
+or
+
+```bash
+kubectl apply -f java-invoker.yaml
+```
+
+## Usage
+
+With the `riff` CLI you need to provide an archive location with `-a`
+and a handler specification with `--handler`. The archive is a jar
+file, which can be shaded with all required dependencies, or it can be
+a Spring Boot fat jar (with dependencies nested in
+`BOOT-INF/lib`). Simple functions that do not require any dependencies
+work just fine. The simplest form of the handler is a class name that
+can be instantiated (with a default constructor). More complex creation scenarios can be handled by giving the handler in the form `<bean>&main=<main>` where
+
+* `<bean>` is a class name or bean name, and
+* `<main` is a Spring `@Configuration` class to create an application context
+
+If you provide a `main` parameter you need to include Spring Boot and
+all the other dependencies of the context in your archive. If the
+`Function` has POJO (i.e. not generic JDK classes) as parameter types,
+then you also need to depend on `spring-cloud-function-context` (and
+inlcude that in the archive).
+
+Example:
+
+```
+riff init java -i greetings -a target/greeter-1.0.0.jar --handler=functions.Greeter
+```
+
+Example with Spring application context
+
+```
+riff init java -i greetings -a target/greeter-1.0.0.jar --handler=greeter&main=functions.Application
```

Comment:
You can also use the handler class instead of the bean name `--handler=functions.Greeter&main=functions.Application`

Reply:
It says that already in the description of the `<bean>` parameter